athenahealth/client.go

```diff
@@ -22,6 +22,9 @@ type Client interface {
 	GetAppointment(ctx context.Context, appointmentID string) (*Appointment, error)
 	ListBookedAppointments(context.Context, *ListBookedAppointmentsOptions) (*ListBookedAppointmentsResult, error)
 	ListChangedAppointments(context.Context, *ListChangedAppointmentsOptions) ([]*BookedAppointment, error)
+	ListAppointmentReasons(ctx context.Context, departmentID, providerID string) ([]*AppointmentReason, error)
+	ListOpenAppointmentSlots(ctx context.Context, departmentID int, opts *ListOpenAppointmentSlotOptions) ([]*OpenAppointmentSlot, error)
```

Comment:
Here dept ID is an int, in the line above it's a string - just double checking

Reply:
@asperkin Types are all over the place with athena - let me clean some of this up where I can.